package.json

```diff
@@ -85,10 +85,15 @@
     "overrides": {
       "jsonpath-plus": "10.0.7",
       "nanoid": "^3.3.8",
-      "katex": "^0.16.21"
+      "katex": "^0.16.21",
+      "openid-client": "5.6.5"
     },
     "patchedDependencies": {
-      "next-auth@4.24.11": "patches/next-auth@4.24.11.patch"
+      "next-auth@4.24.11": "patches/next-auth@4.24.11.patch",
+      "openid-client@5.6.5": "patches/openid-client@5.6.5.patch"
     }
+  },
+  "dependencies": {
+    "follow-redirects": "^1.15.9"
```

Comment:
Can we move this into the web package.json? Or does it need to live in the root package.json?

Reply:
I'll try to play with it today, and update on that @Steffen911

Reply:
@Steffen911 
Unfortunately, I wasn't able to make this work on the web package level. It seems that patches should be handled in the root level of pnpm, otherwise it does not respect that patch.